doc/maintaining/installing/install-from-package.rst

```diff
@@ -5,18 +5,18 @@ Installing CKAN from package
 ============================
 
 This section describes how to install CKAN from package. This is the quickest
-and easiest way to install CKAN, but it requires **Ubuntu 18.04 (Python 2) or 20.04 (Python 3 or Python 2) 64-bit**. If
+and easiest way to install CKAN, but it requires **Ubuntu 18.04 (Python 3.7) or 20.04 (Python 3.8) 64-bit**. If
```

Comment:
Best not to mention Python versions here as Ubuntu 18.04 comes with Python 3.6 (not 3.7)